src/Request/Admin/EditFAQ.js

```diff
@@ -61,45 +79,51 @@ class EditFAQ extends React.Component {
         ...this.state.faq,
         faqCategory: event.target.value,
       },
+      faqCatNull: true,
     }));
   }
+  }
 
   onTitleChange(event) {
-    if (event.target.value == '') {
+    this.setState(() => ({
+      titleNull: false,
+    }))
+    if(event.target.value != ''){
       this.setState(() => ({
-        isSave: false,
         faq: {
           ...this.state.faq,
           title: event.target.value,
         },
-      }));
-    } else {
+    }));
+  } else {
       this.setState(() => ({
-        isSave: true,
         faq: {
           ...this.state.faq,
           title: event.target.value,
         },
-      }));
+        titleNull: true,
+      }))
     }
   }
 
   onContentsChange(event) {
+    this.setState(() => ({
```

Comment:
Could you check it one more?
![image](https://user-images.githubusercontent.com/50673489/114628735-5c6a8400-9c85-11eb-8684-e38d8ffb912f.png)

Also It is not mandantory

Reply:
I produced it works well